src/pages/LocalDetail.tsx

```diff
@@ -0,0 +1,267 @@
+import styled from '@emotion/styled';
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import dfs_xy_conv from '@/components/Translate';
+import {
+  DustState,
+  FineDustState,
+  UltraFineDustState,
+} from '@/components/Dust';
+
+import { LocalDetailType } from '@/type';
+
+const { VITE_WEATHER_API_KEY, VITE_API_KEY, VITE_KAKAO_API_KEY } = import.meta
+  .env;
+
+type Location = {
+  latitude: number;
+  longitude: number;
+};
+
+const LocalDetail = () => {
+  const [temperature, setTemperature] = useState();
+  const [humidity, setHumidity] = useState();
+  const [myLocation, setMyLocation] = useState<Location>({
+    latitude: 0,
+    longitude: 0,
+  });
+  const [forecast, setForecast] = useState();
+  const [forecastImg, setForecastImg] = useState<string[]>([]);
+
+  const date = new Date();
+  const year = date.getFullYear().toString();
+  const month =
+    date.getMonth() < 10
+      ? '0' + (date.getMonth() + 1)
+      : (date.getMonth() + 1).toString();
+  const day = date.getDate() < 10 ? '0' + date.getDate() : date.getDate();
+  const today = year + month + day;
+
+  const currentTime = ('0' + date.getHours()).slice(-2) + '00';
+
+  const location = useLocation();
+  const { stationName, dust, ultraDust, dataTime, dustState }: LocalDetailType =
+    location.state;
+
+  // const fetchLocation = async (stationName: string) => {
+  //   const result = await axios({
+  //     method: 'get',
+  //     url:
+  //       'https://dapi.kakao.com/v2/local/search/address.json?query=' +
+  //       encodeURIComponent(stationName),
+  //     headers: {
+  //       Authorization: `KakaoAK ${VITE_KAKAO_API_KEY}`,
+  //     },
+  //   }).then((res) => res.data.documents[0]);
+
+  //   const { x, y } = result;
+  //   setMyLocation({
+  //     ...myLocation,
+  //     latitude: x,
+  //     longitude: y,
+  //   });
+  //   fetchWeather(myLocation);
+  // };
+
+  // const parser = new XMLParser();
+  const fetchWeather = async (myLocation: Location) => {
+    // code : "toXY"(위경도->좌표, v1:위도, v2:경도),
+    const rs = dfs_xy_conv(
+      'toXY',
+      Number(myLocation.latitude),
+      Number(myLocation.longitude)
+    );
+
+    const { x, y } = rs;
+
+    const result = await axios
+      .get(
+        `http://apis.data.go.kr/1360000/VilageFcstInfoService_2.0/getUltraSrtNcst?serviceKey=${VITE_WEATHER_API_KEY}&numOfRows=10&pageNo=1&dataType=JSON&base_date=${today}&base_time=${currentTime}&nx=${x}&ny=${y}`
+      )
+      .then((res) => res.data);
+
+    setTemperature(result.response.body.items.item[3].obsrValue);
+    setHumidity(result.response.body.items.item[1].obsrValue);
+  };
+
+  const success = ({ coords }: { coords: GeolocationCoordinates }) => {
+    const latitude = coords.latitude; // 위도
+    const longitude = coords.longitude; // 경도
```

Comment:
이거는 변수명에서 충분히 추론 가능할 것 같아서 주석 필요 없다고 생각해요!

```suggestion
    const latitude = coords.latitude;
    const longitude = coords.longitude;
```